### DISCUSSION TITLE
ORIENTATION DEFINED IN NEPER NOT MATCHING IN ABAQUS PROPS

### DISCUSSION THREAD

**Discussion by Anmol-Bakshi** [neperfepx/neper#846] [Q&A] [answered]
I used the following commands to generate 5 grains in neper:
neper -T -n 3 -dim 2 -domain "square(1,1)" -morpho gg -ori "file(orientations.txt)" -oriformat plain -oridescriptor e -o square_2d

neper -M 2d_square.tess -statelset area -rcl 1 -elttype quad -order 1 -interface continuous -format inp -o abq_tet_2d -format msh

where orientations.txt containing phi1, PHI, phi2 are as:
199.1 44.1 59.6
325.1 48.2 33.2
276.2 45.9 60.9
270.8 46.0 68.3
188.7 32.0 69.6

Then, I am converting neper .msh file to abaqus .inp file using a MATLAB code. Then when viewing in ABAQUS PROPS, the angles values are not remaining same. Where am I going wrong?

**Comment by rquey** [comment, marked as answer]
`-ori file(orientations.txt,des=euler-bunge)"` should be utilized to specify the input convention. `-oridescriptor e` only defines the output convention.
